.github/workflows/ci.yaml

```diff
@@ -72,7 +72,7 @@ jobs:
     uses: canonical/data-platform-workflows/.github/workflows/build_charm.yaml@v22.0.0
     with:
       path-to-charm-directory: ${{ matrix.path }}
-      cache: true
+      cache: false
```

Comment:
Should also set `artifact-prefix` to somthing different from `packed-charm-cache`